Description: Drop check for both setuptools and ez_setup
 In Debian we always have setuptools, and it's a build-dep, so if it's missing
 we should hard-fail, not try to work around it by importing ez-setup.
 .
 protobuf (2.6.1-1) unstable; urgency=medium
 .
   * New upstream release.
Author: Robert Edmonds <edmonds@debian.org>
Forwarded: not-needed

--- a/python/setup.py
+++ b/python/setup.py
@@ -7,18 +7,8 @@
 
 # We must use setuptools, not distutils, because we need to use the
 # namespace_packages option for the "google" package.
-try:
-  from setuptools import setup, Extension
-except ImportError:
-  try:
-    from ez_setup import use_setuptools
-    use_setuptools()
-    from setuptools import setup, Extension
-  except ImportError:
-    sys.stderr.write(
-        "Could not import setuptools; make sure you have setuptools or "
-        "ez_setup installed.\n")
-    raise
+from setuptools import setup, Extension
+
 from distutils.command.clean import clean as _clean
 from distutils.command.build_py import build_py as _build_py
 from distutils.spawn import find_executable
